compare_similarity_using_embeddings/main.bal

```diff
@@ -0,0 +1,89 @@
+import ballerina/io;
+import ballerinax/openai;
+import ballerina/lang.'float as langFloat;
+
+configurable string openAIKey = ?;
+
+function vectorNorm(float[] vector1, string norm = "l2") returns float|error {
```

Comment:
Why do you have these functions here? Import the math library and use functions in the math library

Reply:
These functions are temporarily added.